src/main/java/org/breedinginsight/utilities/email/EmailTemplates.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.breedinginsight.utilities.email;
+
+import io.micronaut.context.annotation.Context;
+import org.stringtemplate.v4.ST;
+import org.stringtemplate.v4.STRawGroupDir;
+
+import javax.inject.Singleton;
+import java.io.File;
+
+@Singleton
+@Context
+public class EmailTemplates {
+    private String NEW_ACCOUNT_EMAIL = "newAccountEmail";
+    private File templateDir = new File("src/main/resources/email/");
```

Comment:
This path won't work when running as a compiled jar.  When maven packages up the compiled code into a jar file, it takes everything in `src/main/resources` and copies it to the root of the jar.  Looking at the source code for `STRawGroupDir`, it will first look in the filesystem to see if the path exists, and if it doesn't (which it won't because it's embedded in the jar), will look in the classpath (including the contents of the bi-api jar).  The input into the constructor of `STRawGroupDir` should be `"email/"`

Reply:
Changed and pushed